test/user_test.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'test_helper'
+require_relative '../lib/user'
+####test initialize
+describe "User" do
+  describe "initialize" do
+    before do
+      @user = User.new(slack_id: "USLACKBOT", name: "slackbot", real_name: "Slackbot", status_text: "", status_emoji: "")
+    end
+
+    it "is an instance of User" do
+      expect(@user).must_be_kind_of User
+    end
+  end
+    describe "details" do
+      before do
+        @user = User.new(slack_id: "USLACKBOT", name: "slackbot", real_name: "Slackbot", status_text: "", status_emoji: "")
+      end
+    it "returns user details" do
+    expect(@user.details).must_equal "Slack ID: #{@user.slack_id}, Name: #{@user.name}, Real Name: #{@user.real_name}, Status Text: #{@user.status_text}, Status Emoji: #{@user.status_emoji}"
+  end
+end
```

Comment:
There's actually a missing end here that I think results in unintentionally nested "describe" blocks. Fixing the indentation on line 18 would make that easier to catch.
I use a VSCode plugin called indent-rainbow that I really like to help me catch things like this. It works for ruby and JS!